src/store/pointCache.js

```diff
@@ -8,7 +9,7 @@ export function PointCacheProvider({ children }) {
   const cache = usePointStore();
 
   return (
-    <PointCacheContext.Provider value={cache}>
+    <PointCacheContext.Provider value={useDeepEqualReference(cache)}>
```

Comment:
Using reference equality here causes the `AcceptTransfer` component to rerender for no reason every second.